smac/runhistory/runhistory2epm.py

```diff
@@ -161,7 +163,7 @@ def _build_matrix(self, run_dict: typing.Mapping[RunKey, RunValue],
         """
         raise NotImplementedError()
 
-    def transform(self, runhistory: RunHistory, budget_subset: list = None):
+    def transform(self, runhistory: RunHistory, budget_subset: list = None) -> typing.Tuple[np.ndarray, np.ndarray]:
```

Comment:
Shouldn't budget_subset be also a `typing.list`?

Reply:
Yes, it should.